spec/features/initiative_spec.rb

```diff
@@ -20,4 +20,22 @@
   end
 end
 
+feature 'Visitor edits a initiative' do 
+  background do
+    @initiative = Initiative.create!(:title => 'TestA', :target_date => '01/02/2020')
+  end
+  scenario 'clicks initiative on list view' do
+    visit edit_initiative_path(@initiative)
+    expect(page).to have_content('Edit Initiative')
+  end
+  scenario 'change initiative name and target date' do
+    visit edit_initiative_path(@initiative)
+    fill_in 'Initiative Name', with: 'TestB'
+    fill_in 'Target Date', with: '01/01/2020'
+    fill_in 'Description', with: 'TestJ'
+    click_button 'Submit'
+    expect(page).to have_content('TestB')
```

Comment:
Does it seem like you might want to check for 'TestA' to make sure it's not there?

Reply:
We checked for 'TestA' to make sure it's not there